Formula/lib/libtirpc.rb

```diff
@@ -9,10 +9,23 @@ class Libtirpc < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "91acf8d2991b6c027d780f8662e81f154c6101d651fb80bf16db5e6d8ba0696f"
   end
 
-  depends_on "krb5"
-  depends_on :linux
+  # the build depends are needed while we need to regenerate configure
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  uses_from_macos "krb5"
+  patch :DATA # all patches were sent to libtirpc-devel during the 1.3.6 cycle
```

Comment:
Do you have a link to the thread on the mailing list? Can you add it in a comment?

Reply:
https://sourceforge.net/p/libtirpc/mailman/message/58823787/